fiftyone/core/collections.py

```
@@ -3518,33 +3518,248 @@ def compute_metadata(
             progress=progress,
         )
 
-    def compute_annotation_schema(self, field_name, scan_samples=True):
-        """Compute the annotation schema for a collection's field
+    def generate_label_schemas(self, fields=None, scan_samples=True):
```

Ruff (0.14.8)

Boolean default positional argument in function definition

(FBT002)